### PR TITLE
Simplify Kaggle part in the Bank Marketing Prediction tutorial

### LINKED ISSUE #324: Simplify the Bank Marketing document (Kaggle credentials)
Need to simplify the Kaggle credentials part of the [bank marketing doc](https://github.com/Azure-Samples/azure-ml-federated-learning/blob/main/docs/real-world-examples/bank-marketing.md). Align with the [CCFRAUD doc](https://github.com/Azure-Samples/azure-ml-federated-learning/blob/main/docs/real-world-examples/ccfraud.md) for instance.

### PR DESCRIPTION
**Pull request by thomasp-ms**
## Purpose

* This PR simplifies the Kaggle credentials part of the Bank Marketing Prediction tutorial. It aligns it with the CCFRAUD tutorial. It also slightly improves the general Vertical FL tutorial (typos, wrong links).
* It closes #324 .

## What is the expected review turnaround time?

Urgency:
- [ ] High (needs review today)
- [x] Medium (needs review within a few days - most common)
- [ ] Low (can wait a week)

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## What to Check
* Read the docs, check the links

### CODE REVIEW

File: docs/tutorials/vertical-fl.md [hunk 1]
```diff
@@ -35,22 +36,19 @@ python -m pip install -r ./examples/pipelines/requirements.txt
 ```
 
 ## Data provisioning
-The data format for VFL is different from regular FL. That is why each of our examples contains its own script for uploading data that are needed for a given example.
+The data format for VFL is different from regular FL. That is why each of our examples contains its own script for uploading training data.
```
Comment (majercakdavid) [inline docs/tutorials/vertical-fl.md:39]: I would name it "dataset" or use some more general terminology due to the fact that we split the data into training and validation.

File: docs/tutorials/vertical-fl.md [hunk 2]
```diff
@@ -35,22 +36,19 @@ python -m pip install -r ./examples/pipelines/requirements.txt
...
+Please follow the steps in [CCFRAUD - Run a job to download and store the dataset in each silo](../real-world-examples/ccfraud.md#run-a-job-to-download-and-store-the-dataset-in-each-silo). You will need to have your Kaggle credentials stored in a key vault, as explained in that same page linked above.
 
 
 ### BANK_MARKETING
-
-This example is similar to **CCFRAUD**, however here also host owns part of the feature space. Please follow steps in [BANK_MARKETING - Add your Kaggle credentials to the workspace key vault](../real-world-examples/bank-marketing.md#Add-your-Kaggle-credentials-to-the-workspace-key-vault). Afterwards, follow same steps as for **MNIST** and **please do not forget to replace `--example MNIST_VERTICAL` with `--example BANK_MARKETING_VERTICAL`**).
+This example is similar to **CCFRAUD**, however here the host also owns part of the feature space. Please follow the steps in [BANK_MARKETING - Run a job to download and store the dataset in each silo](../real-world-examples/bank-marketing.md#run-a-job-to-download-and-store-the-dataset-in-each-silo). You will need to have your Kaggle credentials stored in a key vault, as explained in that same page linked above.
```
Comment (majercakdavid) [inline docs/tutorials/vertical-fl.md:48]: What about rest of this part?

Afterwards, follow same steps as for **MNIST** and **please do not forget to replace `--example MNIST_VERTICAL` with `--example BANK_MARKETING_VERTICAL`**).

File: docs/tutorials/vertical-fl.md [hunk 3]
```diff
@@ -35,22 +36,19 @@ python -m pip install -r ./examples/pipelines/requirements.txt
...
+The data format for VFL is different from regular FL. That is why each of our examples contains its own script for uploading training data.
 
-> Note: This will split the data such that each contributor owns its portion of the features and host own only the labels
+> Note: This will split the data such that each contributor owns its portion of the features and the host owns only the labels (except for the Bank Marketing Prediction example).
 
 ### CCFRAUD
-
-Please follow steps in [CCFRAUD - Add your Kaggle credentials to the workspace key vault](../real-world-examples/ccfraud.md#Add-your-Kaggle-credentials-to-the-workspace-key-vault). Afterwards, follow same steps as for **MNIST** and **please do not forget to replace `--example MNIST_VERTICAL` with `--example CCFRAUD_VERTICAL`**).
+Please follow the steps in [CCFRAUD - Run a job to download and store the dataset in each silo](../real-world-examples/ccfraud.md#run-a-job-to-download-and-store-the-dataset-in-each-silo). You will need to have your Kaggle credentials stored in a key vault, as explained in that same page linked above.
```
Comment (majercakdavid) [inline docs/tutorials/vertical-fl.md:44]: What about rest of this part?

Afterwards, follow same steps as for **MNIST** and **please do not forget to replace `--example MNIST_VERTICAL` with `--example CCFRAUD_VERTICAL`**).

File: docs/tutorials/vertical-fl.md (hunk 2 above)
Comment (thomasp-ms) [reply docs/tutorials/vertical-fl.md:48]: this is clearly explained at the location where the link points to - no need to repeat here.

File: docs/tutorials/vertical-fl.md (hunk 3 above)
Comment (thomasp-ms) [reply docs/tutorials/vertical-fl.md:44]: good catch, the link points to instructions that have incorrect parameter value for vertical case

**Review by majercakdavid — APPROVED** [review]

**Pull request merged**